appinventor/appengine/war/Ya.css

```diff
@@ -29,15 +40,15 @@ body, html {
 
 body {
   background-color: lightgrey;
+  font-family: Roboto, Helvetica, Arial, sans-serif;
   /* TODO(opensource): Change to commented version below once we're ready */
   background: url(images/squairy_light.png);
-  font-family: Roboto, Helvetica, Arial, sans-serif;
 }
 
-table td,
-select {
-  font-family: Roboto, Helvetica, Arial, sans-serif;
+.dyslexic * {
+  font-family: OpenDyslexic, Helvetica, Arial, sans-serif;
```

Comment:
Include Roboto as a backup font here so if for some reason OpenDyslexic doesn't load we will fall back to the existing font.